test/e2e/tests/settings/about-metamask/ui-validation.spec.ts

```diff
@@ -0,0 +1,121 @@
+import assert from 'assert';
+import { Suite } from 'mocha';
+import FixtureBuilder from '../../../fixture-builder';
+import {
+  defaultGanacheOptions,
+  unlockWallet,
+  withFixtures,
+} from '../../../helpers';
+import { Driver } from '../../../webdriver/driver';
+
+const selectors = {
+  accountOptionsMenuButton: '[data-testid="account-options-menu-button"]',
+  settingsDiv: { text: 'Settings', tag: 'div' },
+  aboutDiv: { text: 'About', tag: 'div' },
+  metaMaskLabelText: { text: 'MetaMask Version', tag: 'div' },
+  metaMaskVersion: { text: '11.7.3', tag: 'div' },
```

Comment:
Should avoid hard coded 11.7.3 here, test will fail when the version is updated.